### ISSUE TITLE
[Docker] User left waiting with no spinner while supervisord image is pulling

### ISSUE THREAD

**Issue by johnmcollier** [openshift/odo#3033]
/kind bug
/area devfile
/area local

When the supervisord volume and image don't exist on the user's machine, the user is waiting with no spinner:
```
Johns-MacBook-Pro-3:nodejs johncollier$ odo push

Validation
 ✓  Validating the devfile [57137ns]
```

Initially, I thought we wouldn't need a spinner while we pull the supervisord image as it's something in the backend that the user shouldn't worry about. But I think now we need _some_ kind of spinner while the supervisord image is pulled and volume is created (maybe `Initializing component`?) so that the user isn't left waiting.

**Comment by johnmcollier** [comment]
I can work on this

/assign

**Comment by maysunfaisal** [comment]
`Initializing component` sounds good, perhaps put it under the heading `Creating Docker resources for component <name>`

**Comment by johnmcollier** [comment]
This can be closed, since #3120 was merged.
